src/Presentation/Console/Command/RunWorkflowCommand.php

```diff
@@ -48,9 +48,9 @@ class RunWorkflowCommand extends Command
     protected ProjectWorkflow $projectWorkflow;
 
     /**
-     * @var \SprykerSdk\Sdk\Infrastructure\Service\CliValueReceiver
+     * @var \SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver\CliReceiver
      */
-    protected CliValueReceiver $cliValueReceiver;
+    protected CliReceiver $cliValueReceiver;
```

Comment:
It must be depended on the abstraction. Please change it to the `ReceiverInterface` interface.